code/cfile/cfile.cpp

```diff
@@ -634,13 +635,14 @@ void cf_create_directory( int dir_type )
 
 	} while( current_dir != CF_TYPE_ROOT );
 
-	
 	int i;
 
 	for (i=num_dirs-1; i>=0; i-- )	{
 		cf_create_default_path_string( longname, sizeof(longname)-1, dir_tree[i], NULL );
-		mprintf(( "CFILE: Creating new directory '%s'\n", longname ));
-        mkdir_recursive(longname);
+		if( stat( longname, &statbuf ) != 0) {
```

Comment:
I think that stat was not present because I did not notice that the stat.h include was in #ifdef for SCP_UNIX.  It was already used in cf_flush_dir but again I missed that it was #ifdef'd out because I was lazy and editing code in Sublime instead of an IDE.  Moving the include to all platforms should work as according to http://stackoverflow.com/questions/18100097/portable-way-to-check-if-directory-exists-windows-linux-c stat should be available on Windows.

And as I mentioned in the description, I didn't want to waste code complexity on checking something that I was pretty sure has already been checked by the time we get to this point.  If it looks likely that's not the case then I can add that in.